CSS/Styles.css

```diff
@@ -209,17 +264,46 @@ ul {
   border: 2px solid #1a2236;
 }
 
+.shape1 {
+  clip-path: polygon(50% 0%, 100% 50%, 50% 100%, 0% 50%);
+  display: none;
+  width: auto;
+  height: auto;
+  flex: 1 1 20%;
+}
+
+.shape2 {
+  border-radius: 50% 50% 0 50%;
+  display: none;
+  width: auto;
+  height: auto;
+  flex: 1 1 20%;
+}
+
+.yllw {
+  background-color: #febf52;
+}
+
+.teal {
+  background-color: #98dede;
+}
+
 .about {
```

Comment:
[OPTIONAL] I recommend you add the bullet points in your mobile version to make it identical to the template design.

Your design:

<img width="219" alt="Screen Shot 2022-06-03 at 13 25 53" src="https://user-images.githubusercontent.com/82426682/171924168-8249d493-2547-4190-9fa0-90225be9cde6.png">

The template design:

<img width="304" alt="Screen Shot 2022-06-03 at 13 26 15" src="https://user-images.githubusercontent.com/82426682/171924231-f4768fb3-7fde-4d81-98e0-9a7562776aa6.png">

If you decide to implement this suggestion, remember to do so for all three containers.